xpore/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "2.1.1"
+__version__ = "2.2.0"
```

Comment:
no major features added. keep this as 2.1.1

Reply:
I bumped it to 2.2, since the dependency versions changed majorly (numpy/pandas to v2). Can also release it as a 2.1.2 or sth though

Reply:
it is still 2.1.1, the master branch is 2.1.0